src/config/api.rs

```diff
@@ -19,6 +19,10 @@ pub struct Options {
     /// Whether or not to expose the GraphQL playground on the API endpoint.
     #[serde(default = "default_playground")]
     pub playground: bool,
+
+    /// Whether or not the GraphQL endpoint is enabled
+    #[serde(default = "default_graphql")]
```

Comment:
Since this default is not the type-level default, and we depend on that not being changing the serialized form, this needs an additional attribute:
```suggestion
    #[serde(default = "default_graphql", skip_serializing_if = "is_true")]
```
along with an additional function of:
```rust
fn is_true(field: &bool) -> bool {
    *field
}
```
(Normally, you could use just `#[serde(skip_serializing_if = "vector_lib::serde::is_default")]`, but that only works if the default value is the _type_ default, which here it is not. See also https://serde.rs/field-attrs.html#skip_serializing_if

Reply:
Awesome, thanks for the detail! Not super familiar with the serde framework so I appreciate the help!